src/pages/components/split-button.mdx

```diff
@@ -1,10 +1,89 @@
 ---
 title: Split Button
 description: >
-  Description will be here
+  A Split Button is a hybrid between a Dropdown Menu and a Button. It lets users choose from
+  parallel actions and take action on their choice.
+package: '@zendeskgarden/react-buttons'
+components:
+  - buttons/src/elements/SplitButton.tsx
+  - buttons/src/elements/ChevronButton.tsx
+  - buttons/src/elements/Button.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To let users select from multiple parallel actions</li>
+      <li>To reduce visual complexity when there are multiple actions a user can take</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To provide a way to navigate between pages, use
+        <Link to="/components/anchor"> Anchor</Link> instead
```

Comment:
There are spaces within the <Link> component at the moment. I'm having a scenario where if I flow the copy in and use Prettier to format, my local lint says everything is fine. When I push the changes to this branch the build fails because it says there are 104 characters on the line and the limit is 100. It happens when Prettier inserts the spaces using brackets `{' '}`. I did see on Prettiers site that the `printWidth` is a "more of a guidelines than hard rule." [Link](https://prettier.io/docs/en/rationale.html#print-width)

Reply:
Let's figure out a different approach.
- should be able to use `<a>` tags
- no leading spaces on links
- might be best to use relative hrefs (collab with @hzhu for consistency)

Reply:
If the goal is to use markdown, then surround the text with `<Markdown>` and use `[]()` syntax for links. Again, check with @hzhu for consistency.

Reply:
Made this revision. I think it's likely we'll run into the issue I mentioned again though in the future.

Reply:
@rossmoody check out [this thread](https://github.com/zendeskgarden/website/pull/75#discussion_r444548548). JZ figured out the root issue, so feel free to update the `.markdownlintrc` in this PR. Whoever lands the change into `master` first doesn't matter 👍 .

Reply:
@rossmoody check out [this thread](https://github.com/zendeskgarden/website/pull/75#discussion_r444548548). JZ figured out the root issue, so feel free to update the `.markdownlintrc` in this PR. Whoever lands the change into `master` first doesn't matter 👍 .